util-ui/src/main/kotlin/com/tailoredapps/androidutil/ui/keyboard/KeyboardExt.kt

```diff
@@ -0,0 +1,49 @@
+package com.tailoredapps.androidutil.ui.keyboard
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import androidx.appcompat.app.AppCompatActivity
+import io.reactivex.Observable
+
+/**
+ * Created by alexandergrafl on 2019-11-05.
+ */
+
+/**
+ * Creates an instance of [KeyboardStateManager] and returns the [KeyboardStatus] Observable
+ */
+val <A : AppCompatActivity> A.keyboardStates: Observable<KeyboardStatus>
```

Comment:
Have you considered also providing extension functions to fragments for easier discoverability in fragments and less call-site-code (`this.activity.keyboardStates` vs. `this.keyboardStates`)